systemd/postinst.sh

```diff
@@ -27,7 +27,7 @@ echo "Checking Moodle data directory"
 if [ ! -d /opt/moodle/moodledata ]; then
     echo "Creating Moodle data directory"
     mkdir -p /opt/moodle/moodledata
-    chmod 777 /opt/moodle/moodledata
```

Comment:
Why not fix ownership and permissions in one script?

Reply:
Probably best to do it in the container at start

Reply:
You have to make the directory from the host, but set the permissions from within the container in order to have the correct owner on the directory.

Reply:
Good point... do the permissions on the host match the permission inside the container? If not, we can set it in `configure-moodle.sh`